### PR TITLE
Sara/tec 157 reporting to be launched ga week of september 9

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

File: docs/release-notes/july-2024.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ tags:
 
 ### Added
 
-- A new **dashboard** focused on secure guardrails adoption is now in private beta. Find out what percent of findings are fixed before they enter your default or primary branch. To join the private beta, reach out to your Technical Account Manager or Account Executive. See the [Dashboard beta documentation](/semgrep-appsec-platform/dashboard-beta) for more information.
+- A new **dashboard** focused on secure guardrails adoption is now in private beta. Find out what percent of findings are fixed before they enter your default or primary branch. To join the private beta, reach out to your Technical Account Manager or Account Executive. See the [Dashboard beta documentation](/semgrep-appsec-platform/dashboard) for more information.
```
Comment (khorne3) [inline docs/release-notes/july-2024.md:18]:
```suggestion
- A new **dashboard** focused on secure guardrails adoption is now in private beta. Find out what percent of findings are fixed before they enter your default or primary branch. To join the private beta, reach out to your Technical Account Manager or Account Executive. See the [Dashboard documentation](/semgrep-appsec-platform/dashboard) for more information.
```

File: docs/semgrep-appsec-platform/dashboard-old.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
+---
```
Comment (khorne3) [inline docs/semgrep-appsec-platform/dashboard-old.md:1]: If I try to go to this doc, it redirects to the current dashboard doc, right (this is what I see, so I just want to confirm that's correct)? If so, should we delete this doc?

**Review by khorne3 — APPROVED** [review]

File: docs/semgrep-appsec-platform/dashboard-old.md (hunk 2 above)
Comment (s-santillan) [reply docs/semgrep-appsec-platform/dashboard-old.md:1]: Yeah, I temporarily named it dashboard-old for now, and we'll delete it after 1-2 months if it looks like we won't be reverting or keeping any information from this.

File: docs/release-notes/july-2024.md (hunk 1 above)
Comment (s-santillan) [reply docs/release-notes/july-2024.md:18]: Thank you for the catch!

**Pull request merged**